web/src/engine/keyboard-storage/src/cloud/queryEngine.ts

```diff
@@ -328,17 +328,17 @@ export default class CloudQueryEngine extends EventEmitter<EventMap> {
       comma=',';
     }
 
-    let errorStubs: ErrorStub[] = [];
     // Request keyboard metadata from the Keyman Cloud keyboard metadata server
     try {
-      let result: (KeyboardStub|ErrorStub)[]|Error = await this.keymanCloudRequest(cmd,false);
-      return mergeAndResolveStubPromises(result, errorStubs);
+      return await this.keymanCloudRequest(cmd,false);
     } catch(err) {
       // We don't have keyboard info for this ErrorStub
-      console.error(err);
-      let stub: ErrorStub = {error: err};
-      errorStubs.push(stub);
-      return Promise.reject(errorStubs);
+      if(err instanceof Error) {
+        console.error(`Error for request command-string ${cmd}: ${err.message}\n  ${err.stack}`);
+      } else {
+        console.error(err);
+      }
```

Comment:
We shouldn't be emitting messages to console that cannot be suppressed within a library such as KeymanWeb. Generally, callers are responsible for handling errors reported by libraries, and libraries should avoid logging except in debug builds.

I note that the error is bubbled up to the caller with the subsequent `reject()` call.

```suggestion
```

Reply:
Please note the first example report in the description.  Without this `console.error` statement, there would be no informative breadcrumb for that event.  Console statements are our only way (at present) to log breadcrumbs within Keyman Engine for Web when not embedded, as we previously opted out of using `@sentry/minimal` to facilitate doing so.

Are you saying that you would prefer the consumer (for this case, help.keyman.com) log the query details that failed?

Reply:
> Are you saying that you would prefer the consumer (for this case, help.keyman.com) log the query details that failed?

Yes